### PR TITLE
EI-2240 Breaking Change - remove visibility

### PR DESCRIPTION
**Pull request by captnz**
Also EI-2044 Breaking Change - make params consistent

This is just a breaking change in the wrapper code, this new version is backwards compatible with the host hence still >= 6

### CODE REVIEW

**Comment by bernardcooke-iotics** [comment]
Pointing at the API spec on the branch currently - examples all running fine, needs to be updated to point at the released API spec when ready

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 
 ## [Unreleased]
+- BREAKING CHANGE - Removed deprecated visibility
+- BREAKING CHANGE - Changed made parameter names more consistent
```
Comment (bernardcooke-iotics) [inline CHANGELOG.md:10]: Need to be moved to new release heading?

**Review by bernardcooke-iotics — COMMENTED** [review]
Q re: CHANGELOG otherwise LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (captnz) [reply CHANGELOG.md:10]: Cheers I usually do a new PR for the release

**Review by bernardcooke-iotics — APPROVED** [review]

**Pull request merged**